instructions/Instruction.md

```diff
@@ -34,6 +34,7 @@ To open multiple file just select desired files with CTRL key pressed and still
 
 To open folder containing selected file, just click twice on it with right mouse button.
 
+To invert a selection of files, click on a file with the middle mouse button.
```

Comment:
Here should be info that not all records are inverted(all except the clicked one)